### PR TITLE
adding hitemp in radis-app interface

### PR DESCRIPTION
**Pull request by arunavabasu-03**
This pull aims to add the hitemp database in radis-app interface.

### CODE REVIEW

File: frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx [hunk 1]
```diff
@@ -54,7 +61,13 @@ export const MoleculeSelector: React.FC<MoleculeSelectorProps> = ({
...
           variant="standard"
           fullWidth
-          label={isGeisa ? "GEISA 2020 Molecule" : "HITRAN 2020 Molecule"}
+          label={
+            isHitemp
+              ? "Molecule"
+              : isGeisa
+              ? "GEISA 2020 Molecule"
+              : "HITRAN 2020 Molecule"
+          }
```
Comment (erwanp) [inline frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx:70]: just use "Molecule" ? Simpler, clearer and no conditions needed.

File: frontend/src/components/fields/MoleculeSelector/molecules.ts [hunk 2]
```diff
@@ -112,3 +112,13 @@ export const moleculeOptionsGesia = [
...
+export const moleculeOptionsHitemap = [
+  "CO2",
+  "CO",
+  "CH4",
+  "H2O",
+  "NO2",
+  "N2O",
+  "NO",
+  "OH",
+];
```
Comment (erwanp) [inline frontend/src/components/fields/MoleculeSelector/molecules.ts:124]: these are the right HITEMP molecules, but in #627 we did not include "CH4" and "N2O". Can you remove them from the list for the moment ?

**Review by erwanp — CHANGES_REQUESTED** [review]

File: frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx (hunk 1 above)
Comment (arunavabasu-03) [reply frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx:70]: it is perfect I think @erwanp

File: frontend/src/components/fields/MoleculeSelector/molecules.ts [hunk 3]
```diff
@@ -112,3 +112,4 @@ export const moleculeOptionsGesia = [
   "SO3",
   "RUO4",
 ];
+export const moleculeOptionsHitemap = ["CO2", "CO", "H2O", "NO2", "NO", "OH"];
```
Comment (suzil) [inline frontend/src/components/fields/MoleculeSelector/molecules.ts:115]:
```suggestion
export const moleculeOptionsHitemp = ["CO2", "CO", "H2O", "NO2", "NO", "OH"];
```

File: server/main.py [hunk 4]
```diff
@@ -44,7 +44,7 @@ class Payload(BaseModel):
         "transmittance",
         "radiance",
     ]
-    database: Literal["hitran", "geisa"]
+    database: Literal["hitran", "geisa","hitemp"]
```
Comment (suzil) [inline server/main.py:47]:
```suggestion
    database: Literal["hitran", "geisa", "hitemp"]
```

**Review by erwanp — APPROVED** [review]

**Comment by arunavabasu-03** [comment]
I think @suzil we can merge this 😀 .

File: frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx [hunk 5]
```diff
@@ -54,7 +59,13 @@ export const MoleculeSelector: React.FC<MoleculeSelectorProps> = ({
           {...params}
           variant="standard"
           fullWidth
-          label={isGeisa ? "GEISA 2020 Molecule" : "HITRAN 2020 Molecule"}
+          label={
```
Comment (erwanp) [inline frontend/src/components/fields/MoleculeSelector/MoleculeSelector.tsx:62]: Just use Molecule everywhere. No selector. Code will be easier, more minimal result

**Review by erwanp — CHANGES_REQUESTED** [review]

**Review by erwanp — APPROVED** [review]

**Review by suzil — APPROVED** [review]

**Pull request merged**